clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2677,3 +2677,34 @@
       MatchMode: Full
       In: OpenMP::OpenMP_CXX
       Out: ""
+
+- Rule: rule_CUDA_PROPAGATE_HOST_FLAGS
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: CUDA_PROPAGATE_HOST_FLAGS
+  In: ${func_name}(${value})
+  Out: ${func_name}(${value})
+  Subrules:
+    value:
+      MatchMode: Full
+      In: CUDA_PROPAGATE_HOST_FLAGS
+      Out: DUMP_VAR
+
+- Rule: rule_CUDA_USE_STATIC_CUDA_RUNTIME
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: CUDA_USE_STATIC_CUDA_RUNTIME
+  In: ${func_name}(${value})
+  Out: ${func_name}(${value})
+  Subrules:
+    value:
+      MatchMode: Full
+      In: CUDA_USE_STATIC_CUDA_RUNTIME
+      Out: DUMP_VAR
+
+- Rule: rule_cuda_source_property_format
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: set_source_files_properties_cuda_source_property_format
+  In: set_source_files_properties${empty}(${src_files} PROPERTIES${some_other_prop}CUDA_SOURCE_PROPERTY_FORMAT OBJ${other_props})
```

Comment:
Have you verified that removing "CUDA_SOURCE_PROPERTY_FORMAT OBJ" does NOT impact the build process for the migrated code?

Reply:
Yes